### PR TITLE
Use Consistent name for Redis

### PR DESCRIPTION
**Pull request by aashishpsaini**
### Motivation / Background

While working in the codebase, found that we are not using Redis name consistently across the repo. For the places we are referring to gem `redis` it makes sense to use smallcase `redis` whereas in case where we are referring to the `Redis` library, I think we should use consistent upcase `Redis`

### Detail

This Pull Request updates the name where we are referring `Redis` library to use consistent upcase.

### Checklist

Before submitting the PR make sure the following are checked:

* [x] This Pull Request is related to one change. Changes that are unrelated should be opened in separate PRs.
* [x] Commit message has a detailed description of what changed and why. If this PR fixes a related issue include it in the commit message. Ex: `[Fix #issue-number]`
* [x] Tests are added or updated if you fix a bug or add a feature.
* [x] CHANGELOG files are updated for the changed libraries if there is a behavior change or additional feature. Minor bug fixes and documentation changes should not be included.

### CODE REVIEW

File: activejob/test/support/integration/adapters/sidekiq.rb [hunk 1]
```diff
@@ -9,7 +9,7 @@ module SidekiqJobsManager
   def setup
     ActiveJob::Base.queue_adapter = :sidekiq
     unless can_run?
-      puts "Cannot run integration tests for sidekiq. To be able to run integration tests for sidekiq you need to install and start redis.\n"
+      puts "Cannot run integration tests for sidekiq. To be able to run integration tests for sidekiq you need to install and start Redis.\n"
```
Comment (p8) [inline activejob/test/support/integration/adapters/sidekiq.rb:12]: I think we should probably capitalize Sidekiq as well:
```suggestion
      puts "Cannot run integration tests for Sidekiq. To be able to run integration tests for Sidekiq you need to install and start Redis.\n"
```

**Comment by p8** [comment]
If we're going to update this we probably also need to update:

* `backburner` to `Backburner` (beanstalkd is correct)
https://github.com/rails/rails/blob/820fbd9102a5488f3bfcc91ec35324b7076cc83e/activejob/test/support/integration/adapters/backburner.rb#L11
* `queue_classic` to `QueueClassic` and `postgresql` to `PostgreSQL`
https://github.com/rails/rails/blob/820fbd9102a5488f3bfcc91ec35324b7076cc83e/activejob/test/support/integration/adapters/queue_classic.rb#L43
* `resque` to `Resque` and `redis` to `Redis`:
https://github.com/rails/rails/blob/820fbd9102a5488f3bfcc91ec35324b7076cc83e/activejob/test/support/integration/adapters/resque.rb#L9
* `sneakers` to `Sneakers` and `rabbitmq` to `RabbitMQ`:
https://github.com/rails/rails/blob/820fbd9102a5488f3bfcc91ec35324b7076cc83e/activejob/test/support/integration/adapters/sneakers.rb#L20

File: activejob/test/support/integration/adapters/sidekiq.rb (hunk 1 above)
Comment (aashishpsaini) [reply activejob/test/support/integration/adapters/sidekiq.rb:12]: Thanks @p8, will do it.

**Comment by jonathanhefner** [comment]
I pushed a commit to [squash the commits](https://guides.rubyonrails.org/contributing_to_ruby_on_rails.html#squashing-commits).

Thank you, @aashishpsaini! :heavy_check_mark: And thank you, @p8! :raised_hands:

**Pull request merged**